client/app.vue

```diff
@@ -1,98 +1,24 @@
 <template>
```

Comment:
Previously a lot of template stuff was in the `app.vue` but I've moved it to a `layouts/default.vue` so that we can optionally hide it.